addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/timing/OnlineSimpleLinearRegression.java

```diff
@@ -32,6 +32,8 @@
  * <br>
  * By convention, we fix correlation and slope at 1.0 and the intercept at 0.0 when insufficient
  * data points (<2) have been added.
+ *
+ * @since 1.19.0
```

Comment:
Sorry I wasn't clear. 1.18.0 is out so this will be upcoming 1.19.0.

Reply:
done